homeassistant/components/awair/translations/en.json

```diff
@@ -24,12 +27,28 @@
                 },
                 "description": "Please re-enter your Awair developer access token."
             },
-            "user": {
+            "local" :{
+                "data": {
+                    "host": "Hostname or IP address of the device"
+                },
+                "description": "Awair Local API must be enabled following these steps: https://support.getawair.com/hc/en-us/articles/360049221014-Awair-Element-Local-API-Feature"
+            },
+            "cloud" :{
                 "data": {
                     "access_token": "Access Token",
                     "email": "Email"
                 },
                 "description": "You must register for an Awair developer access token at: https://developer.getawair.com/onboard/login"
+            },
+            "discovery_confirm": {
+                "description": "Do you want to setup local {model} device: {host}?"
+            },
+            "user": {
+                "menu_options": {
+                    "cloud": "Cloud API",
+                    "local": "Local API"
```

Comment:
Drop word "API". Not user friendly. 

- Connect locally (preferred)
- Connect via the cloud